ui/app/templates/storage/volumes/dynamic-host-volume.hbs

```diff
@@ -0,0 +1,117 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{#each this.breadcrumbs as |crumb|}}
+  <Breadcrumb @crumb={{crumb}} />
+{{/each}}
+{{page-title "Dynamic Host Volume " this.model.name}}
+<section class="section with-headspace">
+  <h1 class="title" data-test-title>{{this.model.name}}</h1>
+
+  <div class="boxed-section is-small">
+    <div class="boxed-section-body inline-definitions">
+      <span class="label">Volume Details</span>
+      {{#if this.system.shouldShowNamespaces}}
+        <span class="pair" data-test-volume-namespace>
+          <span class="term">Namespace</span>
+          {{this.model.namespace}}
+        </span>
+      {{/if}}
+      <span class="pair" data-test-volume-node>
+        <span class="term">Client</span>
+        {{this.model.node.name}}
+      </span>
+      <span class="pair" data-test-volume-plugin>
+        <span class="term">Plugin</span>
+        {{this.model.pluginID}}
+      </span>
+      <span class="pair" data-test-volume-create-time>
+        <span class="term">Create Time</span>
+        <span class="tooltip" aria-label="{{format-month-ts this.model.createTime}}">
+          {{moment-from-now this.model.createTime}}
+        </span>
+      </span>
+      <span class="pair" data-test-volume-modify-time>
+        <span class="term">Modify Time</span>
+        <span class="tooltip" aria-label="{{format-month-ts this.model.modifyTime}}">
+          {{moment-from-now this.model.modifyTime}}
+        </span>
+      </span>
+      {{#if this.model.capacityBytes}}
+        <span class="pair" data-test-volume-capacity>
+          <span class="term">Capacity</span>
+          {{format-bytes this.model.capacityBytes}}
+        </span>
+      {{/if}}
+    </div>
+  </div>
+
+  <div class="boxed-section">
+    <div class="boxed-section-head">
+      Allocations
+    </div>
+    <div class="boxed-section-body {{if this.model.allocations.length "is-full-bleed"}}">
+      {{#if this.sortedAllocations.length}}
+        <ListTable
+          @source={{this.sortedAllocations}}
+          @class="with-foot" as |t|>
+          <t.head>
+            <th class="is-narrow"><span class="visually-hidden">Driver Health, Scheduling, and Preemption</span></th>
```

Comment:
I don't see this column in the preview. This shows up when the allocations are in some particular condition?

Aside / for my education: I think this table is the same one we have for CSI write/read allocs but not the component we have on the job details? Is there a shared component to extract here or is that messy to do?

Reply:
It's technically the same column (on client detail page, too, etc.) but there's a "context" prop that we pass a string to, and that determines content.

https://github.com/hashicorp/nomad/blob/dfce76e5114a939b5b00c64600f016a24afb0f36/ui/app/templates/components/allocation-row.hbs#L47

Reply:
Thanks for the context and education! :grin: